api/v1beta1/v1beta2_condition_consts.go

```diff
@@ -198,4 +194,19 @@ const (
 
 	// InspectionFailedV1Beta2Reason applies to a condition when inspection of the underlying object failed.
 	InspectionFailedV1Beta2Reason = "InspectionFailed"
+
+	// WaitingForClusterInfrastructureV1Beta2Reason documents an infra Machine waiting for the cluster
+	// infrastructure to be ready.
+	WaitingForClusterInfrastructureV1Beta2Reason = "WaitingForClusterInfrastructure"
```

Comment:
```suggestion
	WaitingForClusterInfrastructureReadyV1Beta2Reason = "WaitingForClusterInfrastructureReady"
```

I think that's a bit clearer and also consistent with WaitingForControlPlane**Initialized**

(Just want to make sure these are fine, as I assume we would also use the same in production infra providers. For CAPDev it wouldn't matter too much of course)